src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -52,8 +49,7 @@ private static void setupPresetTests(Application application) {
      */
     private static void setupCommandTests(Application application) {
         application.addTest("Load secret command", new SelectLoadSecretCommandOptionListener());
-        ICompoundCommand command = new TriangleFigureComplexCommand().createCompoundCommand();
-        CommandsFeature.getDriverCommandManager().setCurrentCommand(command);
+        application.addTest("Load compound command", new SelectLoadCompoundCommandOptionListener());
```

Comment:
This is a specific type of a command (triangle). Please add this to description)